### ISSUE TITLE
15002 endpoint

### ISSUE THREAD

**Issue by nykfank** [AlCalzone/node-tradfri-client#106]
I saw in endpoints.ts that you found an endpoint called hsAccessoryLink with the code 15002. I failed to access this one though the coap-client, while all the other ones worked fine. Is there something special for this endpoint to make it work? I am interested in it because it might help with this question. I was wondering how to detect if a bulb in IKEA trådfri is "Unreachable"? This is the case when the bulb has it's power turned off. I tried to read this state using the API of the gateway through COAP, but there was no change in bulb status after turning off is power. I hoped this would be possible though the 15002 endpoint.

**Comment by AlCalzone** [comment]
Having 15002 in endpoints is a red herring. Back when I tried to started developing this, I copied constants from the decompiled app around and tried to make sense of them. `hsAccessoryLink` is a sub property of groups, which links groups and the devices in it: https://github.com/AlCalzone/node-tradfri-client/blob/97a7bc13a7918da49cafc14fd4750f5d5a6b27a0/src/lib/group.ts#L118

Regarding your other question, https://github.com/AlCalzone/node-tradfri-client/issues/93 has some answers.
Basically the gateway assumes a device is "alive" until it stops responding to requests. You can try to force a check by sending a slightly changed value, e.g. a change of the brightness by +/- 1. If a bulb it offline, this should result in the alive state becoming false after a while.

**Comment by nykfank** [comment]
Thanks again, I've got it working now. Just realized the alive property is just the 9019 code in the device endpoint. I get reliable results by changing brightness by 1, waiting for 10 seconds, then checking alive.

**Comment by nykfank** [comment]
Better even to change warmth by 1 (and back) each seconds for 5 seconds. This will not turn on the bulb if it was alive and turned off and still detects alive status reliably.

**Comment by AlCalzone** [comment]
Good idea! I think this should be documented. Would you be willing to describe your process briefly #107?
